app/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id 'application'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
```

Comment:
Dependencies should have an order. It is important for each project because it will grow during the time, and you can have mess.

Order should be like that (it is not rule, it is recommendation):
- annotation processors
- compile
- implementation
- tests

Reply:
Please, fix that one .